clash-prelude/src/Clash/Prelude/BlockRam.hs

```diff
@@ -901,5 +900,5 @@ trueDualPortBlockRam inA inB =
   -- ^ Outputs data on /next/ cycle. When writing, the data written
   -- will be echoed. When reading, the read data is returned.
 trueDualPortBlockRam wmA wmB inA inB =
-  E.trueDualPortBlockRam wmA wmB hasClock hasClock inA inB
+  E.trueDualPortBlockRam E.tdpDefault hasClock hasClock inA inB
```

Comment:
This shouldn't even compile, both before and after the commit? The implicit version doesn't get `wmA` and `wmB`. CPP mucking stuff up again, I bet you only built with multiple hidden :-).